app/models/card.rb

```diff
@@ -1,7 +1,23 @@
 class Card < ActiveRecord::Base
-  before_create :set_review
+  before_validation :set_review_date, on: :create
+  validate :original_translated_text_equal
+  validates :original_text, :translated_text, :review_date,
+            presence: { message: 'Необходимо заполнить поле.' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.